openfisca_france/param/param.xml

```diff
@@ -5286,6 +5286,9 @@
       <CODE code="age_min" description="Âge minimal requis" format="integer" type="age">
         <VALUE deb="2002-01-01" fin="2014-03-31" valeur="65" />
       </CODE>
+      <CODE code="taux_invalidite_aspa_anticipe" description="Taux d'invalidité nécessaire pour percevoir l'ASPA de manière anticipée">
+        <VALUE deb="2002-01-01"  fin="2015-12-31" valeur="50" />
```

Comment:
@fpagnoux percent of invalidity should be written using decimals. You also need to inclue an attribute format="percent" for this parameter.